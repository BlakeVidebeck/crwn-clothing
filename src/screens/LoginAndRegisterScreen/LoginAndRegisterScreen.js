import React from 'react'

import Login from '../../components/login/Login'
import Register from '../../components/register/Register'

import { LoginAndRegisterContainer } from './LoginAndRegisterScreenStyles'

const LoginAndRegisterScreen = () => {
	return (
		<LoginAndRegisterContainer>
			<Login />
			<Register />
		</LoginAndRegisterContainer>
	)
}

export default LoginAndRegisterScreen
