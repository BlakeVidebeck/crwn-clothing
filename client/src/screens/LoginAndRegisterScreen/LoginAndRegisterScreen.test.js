import React from 'react'
import { shallow } from 'enzyme'
import LoginAndRegisterScreen from './LoginAndRegisterScreen'

it('should render LoginAndRegisterScreen component', () => {
	expect(shallow(<LoginAndRegisterScreen />)).toMatchSnapshot()
})
