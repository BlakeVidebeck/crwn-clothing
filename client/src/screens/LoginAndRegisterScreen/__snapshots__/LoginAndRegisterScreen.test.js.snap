// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render LoginAndRegisterScreen component 1`] = `
<styled.div>
  <Connect(Login) />
  <Connect(Register) />
</styled.div>
`;
